src/blockchain/log-processors/token/mint.ts

```diff
@@ -15,13 +18,19 @@ export function processMintLog(db: Knex, augur: Augur, trx: Knex.Transaction, lo
     blockNumber:     log.blockNumber,
   }).into("transfers").asCallback((err: Error|null): void => {
     if (err) return callback(err);
-    increaseTokenBalance(db, augur, trx, log.token || log.address, log.target, Number(log.amount || log.value), callback);
+    parallel([
+      (next: AsyncCallback): void => increaseTokenSupply(db, augur, trx, log.token || log.address, Number(log.amount || log.value), next),
+      (next: AsyncCallback): void => increaseTokenBalance(db, augur, trx, log.token || log.address, log.target, Number(log.amount || log.value), next),
```

Comment:
`log.amount || log.value` is referenced 3 times, it's probably worth a variable. Probably all those solidity stack depth limitation issues you had to deal with ;)

Reply:
Haha, yea will do.